db/migrate/20171004220124_create_trips.rb

```diff
@@ -8,7 +8,7 @@ def change
       trip.integer   :end_station_id
       trip.integer   :bike_id
       trip.text      :subscription_type
-      trip.integer   :zip_code
+      trip.bigint  :zip_code
```

Comment:
Believe this should be a string. `bigint` will convert a zip like `04005` into `4005`, which is not the behavior I want.

Sometimes I think about it like this: am I ever going to perform addition on this number? Multiply it? If the answer is no, then I would start to consider storing it as a string. If it can potentially start with a 0, I almost definitely want to store it as a string.